diff --git a/dist/harmony/template/app.js b/dist/harmony/template/app.js
index 8d3c3dccfed8f97c9a0ae092512a9a443f499a82..13f1747f5de511e326d4d802cfa0269c82c22652 100644
--- a/dist/harmony/template/app.js
+++ b/dist/harmony/template/app.js
@@ -102,7 +102,7 @@ class Parser extends base_1.default {
   }
 
   onMemoryLevel(level: AbilityConstant.MemoryLevel) {
-    let levelRes: number
+    let levelRes: number = 0
 
     switch (level) {
       case AbilityConstant.MemoryLevel.MEMORY_LEVEL_MODERATE:
